applications/swimming_DEM_application/custom_utilities/fields/pouliot_flow_field_2D.cpp

```diff
@@ -15,16 +15,16 @@ void PouliotFlowField2D::ResizeVectorsForParallelism(const int n_threads)
 void PouliotFlowField2D::UpdateCoordinates(const double time, const array_1d<double, 3>& coor, const int i_thread)
```

Comment:
Would you please improve the styling and naming for the arguments? Time, Coordinates, ThreadNumber, ...

Reply:
I use uncapitalized names in general in my application... I can change this but I would prefer to do it in a planned, organized way. Likewise I use the abbreviation coor quite often. This is also a change that I could do everywhere at once but not in this pull request, if you don't mind.